samples/options_builder.py

```diff
@@ -1,4 +1,5 @@
 import sys
```

Comment:
This file shouldn't be in the commit for this, right? I think it's part of Options Builder